cmd/dep/status.go

```diff
@@ -486,6 +486,8 @@ func runStatusAll(ctx *dep.Ctx, out outputter, p *dep.Project, sm gps.SourceMana
 				if pp, has := p.Manifest.Ovr[proj.Ident().ProjectRoot]; has && pp.Constraint != nil {
 					bs.hasOverride = true
 					bs.Constraint = pp.Constraint
+				} else if pp, has := p.Manifest.Constraints[proj.Ident().ProjectRoot]; has && pp.Constraint != nil {
```

Comment:
The comment above this block talks about checking in Override. Since with this we check Constraints separately, let's add a comment about that too.